cmd/tuple/delete.go

```diff
@@ -32,13 +33,53 @@ var deleteCmd = &cobra.Command{
 	Short:   "Delete Relationship Tuples",
 	Long:    "Delete relationship tuples from the store.",
 	Example: "fga tuple delete --store-id=01H0H015178Y2V4CX10C2KGHF4 user:anne can_view document:roadmap",
-	Args:    cobra.ExactArgs(3), //nolint:gomnd
```

Comment:
because we need 3 args OR the --file i created a custom function for it to be used on the `Args` field: https://github.com/openfga/cli/blob/0b0ec8d217c03edfe0863e50a1ee7e7727826642/cmd/tuple/delete.go#L108-L116

Reply:
Thank you!